src/main/java/calculator/stringCal.java

```diff
@@ -0,0 +1,60 @@
+package calculator;
+
+public class stringCal {
+
```

Comment:
자바 컨벤션에서 클래스는 파스칼케이스로 작성합니다
축약어 쓰면 코드 이해하기가 어려워집니다 다음 미션에서 신경쓰면 좋겠네요

```suggestion

public class StringCalculator {

```